src/Pipeline.jl

```diff
@@ -187,6 +188,7 @@ function get_ref_stats_kwargs(ref_config::Dict{Any, Any}, reg_config::Dict{Any,
         :tikhonov_mode => tikhonov_mode,
         :dim_scaling => dim_scaling,
         :model_errors => model_errors,
+        :Δt => time_shift,
```

Comment:
Maybe it would be better to name it `time_shift` instead of `Δt` so that for sure it will not be confused with model time step?

```suggestion
        :time_shift => time_shift,
```